packages/blockchain-common/test/registries/FileRegistry.test.ts

```diff
@@ -32,6 +28,7 @@ chai.use(chaiAsPromised);
 describe('FileRegistry', () => {
 
     const testFabricRegistryName: string = 'test';
+    const testRegistriesPath: string = path.join(__dirname, '../tmp/registries');
```

Comment:
`path.join(__dirname, '../tmp/registries');` should be `path.join(__dirname, '..', 'tmp', 'registries')`